app/javascript/components/enrollment/steps/Identification.js

```diff
@@ -1,19 +1,27 @@
 import React from 'react';
 import { PropTypes } from 'prop-types';
 import { Button, Card, Col, Form } from 'react-bootstrap';
+import Select from 'react-select';
+
 import * as yup from 'yup';
 import moment from 'moment-timezone';
-import Select from 'react-select';
 
 import DateInput from '../../util/DateInput';
 import InfoTooltip from '../../util/InfoTooltip';
-import supportedLanguages from '../../../data/supportedLanguages.json';
+import { convertLanguageCodesToNames, getLanguageSupported, getLanguagesAsOptions } from '../../../utils/Languages';
 
 const WORKFLOW_OPTIONS = [
   { label: 'Exposure (contact)', value: 'exposure' },
   { label: 'Isolation (case)', value: 'isolation' },
 ];
 
+const cursorPointerStyle = {
```

Comment:
Can we move this to a css file?

Reply:
Im not sure, but i'll look into it

Reply:
could use this class or something: https://github.com/SaraAlert/SaraAlert/blob/master/app/javascript/packs/stylesheets/custom_table.scss#L18

Reply:
but move it to a more generic file that's not just custom table

Reply:
Removed, and replaced with direct theme'ing